src/flag_evaluation/error_code.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class ErrorCode(Enum):
```

Comment:
> Requirement 1.4.7
> In cases of abnormal execution, the evaluation details structure's error code field MUST contain a string identifying an error occurred during flag evaluation and the nature of the error.

String is specified for this type, fyi.

Reply:
I've changed all the enums to strings now